FAQ.md

```diff
@@ -0,0 +1,8 @@
+---

```

Comment:
The file path for FAQ.md file is moved.  Please move back under _faq folder.

Reply:
Done.